lendingbot.py

```diff
@@ -187,11 +117,8 @@
 	loadedFiles = config.read([config_location])
 	# Create default config file if not found
```

Comment:
Look at the total files changed instead of a single commit, I think that is why you are seeing the comment like that. (Because it currently says copy)